### ISSUE TITLE
Vrapper Swing Component

### ISSUE THREAD

**Issue by gungwald** [vrapper/vrapper#870]
Can vrapper be converted into a Swing component like JTextArea that anyone can use in their Swing app?

**Comment by albertdev** [comment]
No, Vrapper depends entirely on Eclipse's JFace and SWT, so I expect you to have considerable trouble to port it.

If you still want something for a Swing app, maybe check JavaVi. It's a Vi emulator for NetBeans, but according to the project page it is built on `java.swing.text`: https://sourceforge.net/projects/jvi/

**Comment by albertdev** [comment]
I've edited my comment above a few times, but the short answer is that it's just a **lot** of work, and there is no plan to do it.
